.github/workflows/auto-upgrade.yaml

```
@@ -0,0 +1,19 @@
+name: Auto upgrade the Helm Chart
+
+on:
+  schedule:
+    - cron: "0 8 * * *"
+  workflow_dispatch: {}
+
+jobs:
+  run_upgrade:
+    name: Run the upgrade script
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v5.0.0
```

Best practice would be to pin the version to a SHA

```suggestion
      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
```